landing-page/content/common/talks.md

```diff
@@ -22,6 +22,9 @@ url: talks
 
 Here is a list of talks and other videos related to Iceberg.
 
+### [Why You Shouldn’t Care About Iceberg | Tabular](https://www.youtube.com/watch?v=_GW3GYZK66U)
+**Date**: May 4, 2022, **Authors**: Ryan Blue
```

Comment:
This date isn't correct, right? The talk was 24 March

Reply:
Fixed! Sorry I used the date the video was published (yesterday)